static/widgets/site-templates-widget.ts

```diff
@@ -75,31 +101,72 @@ class SiteTemplatesWidget {
         const first = Object.entries(templatesConfig.templates)[0][0]; // preview the first entry
         this.img.src = this.getAsset(first.replace(/[^a-z]/gi, ''));
     }
-    async populate() {
+    populateUserTemplates() {
+        const userTemplates: Record<string, UserSiteTemplate> = JSON.parse(local.get('userSiteTemplates', '{}'));
+        const userTemplatesList = $('#site-user-templates-list');
+        userTemplatesList.empty();
+        if (Object.entries(userTemplates).length === 0) {
+            userTemplatesList.append(`<span>Nothing here yet</span>`);
+        } else {
+            for (const [id, {title, data}] of Object.entries(userTemplates)) {
+                userTemplatesList.append(
+                    `<li>` +
+                        `<div class="title" data-data="${data}">${title}</div>` +
```

Comment:
I don't know how this compares to the site tempaltes but... it's kinda scary to have user input "raw"ly placed in tags here.

I know it's "only" the user's _own_ computer but... this sometimes is a security consideration even so.

Is there a way to sanitise these before we add them here? Maybe I'm being FUDdish and ignoring the 9,999 other incarnations of this problem elsewhere....

Reply:
Ah, thank god for reviews because this did completely slip my mind 😄 I figure we may as well html escape it for good practice sake